src/components/scrollToTop/ScrollToTop.tsx

```diff
@@ -6,7 +6,8 @@ const ScrollToTop = () => {
   const routePath = useLocation()
 
   useEffect(() => {
-    () => window.scrollTo(0, 0)
+    window.scrollTo(0, 0)
+    behavior: 'smooth'
```

Comment:
Интересно, как это должно работать? В документации вроде немного иначе: 
```suggestion
    window.scrollTo({
      top: 0,
      left: 0,
      behavior: 'smooth'
    });
```

Reply:
да, так гораздо лучше)